overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -61248,7 +61251,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "This is how kids movies are made",
-          "desc:8": "The Wand of Animation is a Wand-erous tool that lets you turn any block alive... Not that you §oneed§r to do that, but it\u0027s fun!"
+          "desc:8": "The Wand of Animation is a Wand-erous tool that lets you turn any block alive, and lets you powered spawn it after... Not that you §oneed§r to do that, but it\u0027s fun!"
```

Comment:
It doesnt let you spawn it after unless you just want stone blocks, since the rftools mob duplicator does not have spawn exact mode on